src/commands/task.ts

```diff
@@ -82,7 +82,8 @@ export default class TaskExec extends Command {
     let service_name;
     const slug = ServiceVersionSlugUtils.build(parsed_slug.component_account_name, parsed_slug.component_name, args.task, parsed_slug.tag);
     const ref = Refs.safeRef(slug);
```

Comment:
Use `ComponentConfig.getNodeRef(slug)` that should match 1:1 with the compose.services keys